suite-common/formatters/src/utils/convertCryptoToFiatAmount.ts

```diff
@@ -0,0 +1,25 @@
+import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { FiatRates } from '@suite-common/wallet-types';
+import { FiatCurrencyCode } from '@suite-common/suite-config';
+
+type CryptoToFiatAmountValue = string | null;
+
+export const convertCryptoToFiatAmount = ({
+    value,
+    rates,
+    fiatCurrency,
+    network,
+}: {
+    value: CryptoToFiatAmountValue;
+    rates?: FiatRates;
```

Comment:
Because fiat rates are result of find method of coins selector throughout the code base so they can possibly be undefined even though this shouldn't really happen but we cannot guarantee it.

The formatting method counts with this and accepts it as undefined as well.

Reply:
Why should that function care about something like that? It's standalone function that doesn't care about redux/selectors or anything else form outside world. It needs rates to work correctly so it should have this param mandatory, if you don't have rates it doesn't make sense to call it at all.

Delegate responsibility to caller.

Reply:
fixed